api/handlers_ltm.go

```diff
@@ -107,16 +107,23 @@ func (s *server) ModifyClientSSLProfile(w http.ResponseWriter, r *http.Request)
 		client: ltmService,
 	}
 
-	if err := orch.modifyClientSSLProfile(r.Context(), name, &data); err != nil {
+	var out string
+	if err := orch.modifyClientSSLProfile(r.Context(), &data); err != nil {
 		handleError(w, err)
 		return
+	} else {
+		out = fmt.Sprintf("modified client-ssl profile %s on host %s", name, host)
 	}
```

Comment:
you don't need the else here since you're returning in the if, and then you can also dump the `var out string`.  json.Marshal will actually just marshal your string into a string (which is valid JSON surprisingly), so probably you can skip the `Marshal` all together and just keep the previous changes

```golang
	w.Header().Set("Content-Type", "application/json")
	w.WriteHeader(http.StatusOK)
	w.Write([]byte(out))
```